README.md

```diff
@@ -258,6 +258,10 @@ environment variable:
   this pointer. Note that it is not currently guaranteed that code that works
   with `-Zmiri-track-raw-pointers` also works without
   `-Zmiri-track-raw-pointers`, but for the vast majority of code, this will be the case.
+* `-Zmiri-measureme=<name>` enables `measureme` profiling for the interpreted program.
+   This can be used to find which parts of your program are executing slowly under Miri.
+   The profile is written out to a file with the prefix `<name>`, and can be processed
+   using the tools in the repository https://github.com/rust-lang/measureme
```

Comment:
Please keep the option list alphabetically sorted ("measureme" should be before "track").
```suggestion
   using the tools in the repository https://github.com/rust-lang/measureme.
```